packages/core/package.json

```diff
@@ -103,7 +103,7 @@
   },
   "scripts": {
     "build": "coffee -b -o lib src",
-    "pretest": "npm build",
+    "pretest": "npm run build",
```

Comment:
This shall have been in a separate branch/commit/pull request

Reply:
OK, separating out and will update all of them.